components/dot_menu/post_reminder_submenu.tsx

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useDispatch} from 'react-redux';
+
+import {FormattedMessage, FormattedDate, FormattedTime, useIntl} from 'react-intl';
+import {ChevronRightIcon, ClockOutlineIcon} from '@mattermost/compass-icons/components';
+
+import * as Menu from 'components/menu';
+import {getCurrentMomentForTimezone} from 'utils/timezone';
+import {openModal} from 'actions/views/modals';
+import {ModalIdentifiers} from 'utils/constants';
+import {toUTCUnix} from 'utils/datetime';
+import PostReminderCustomTimePicker from 'components/post_reminder_time_picker_modal';
+import {addPostReminder} from 'mattermost-redux/actions/posts';
+import {t} from 'utils/i18n';
+
+import {Post} from '@mattermost/types/posts';
+
+type Props = {
+    userId: string;
+    post: Post;
+    isMilitaryTime: boolean;
+    timezone?: string;
+}
+
+const postReminderTimes = [
+    {id: 'thirty_minutes', label: t('post_info.post_reminder.sub_menu.thirty_minutes'), labelDefault: '30 mins'},
+    {id: 'one_hour', label: t('post_info.post_reminder.sub_menu.one_hour'), labelDefault: '1 hour'},
+    {id: 'two_hours', label: t('post_info.post_reminder.sub_menu.two_hours'), labelDefault: '2 hours'},
+    {id: 'tomorrow', label: t('post_info.post_reminder.sub_menu.tomorrow'), labelDefault: 'Tomorrow'},
+    {id: 'custom', label: t('post_info.post_reminder.sub_menu.custom'), labelDefault: 'Custom'},
+];
```

Comment:
nit (feel free to ignore)
But if you bring this inside of the component then you can use formatMessage directly instead of t function.

And further down it would become just 
```
const labels = label
```

Reply:
Not changing this for now